docs/user-event/install.mdx

```diff
@@ -0,0 +1,36 @@
+---
+id: install
+title: Installation
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+<Tabs defaultValue="npm" values={[
+    { label: 'npm', value: 'npm' },
+    { label: 'Yarn', value: 'yarn' },
+]}>
+<TabItem value="npm">
+
+```sh
+npm install --save-dev @testing-library/user-event@^14.0.0-beta
+```
+
+</TabItem>
+<TabItem value="yarn">
+
+```sh
+yarn add --dev @testing-library/user-event@^14.0.0-beta
+```
+
+</TabItem>
+</Tabs>
+
+Note that `@testing-library/user-event` requires `@testing-library/dom`.
+
+If you use one of the
+[framework wrappers](../dom-testing-library/install#wrappers), it is important
```

Comment:
I'm not sure if this comment will confuse people or help. Did you see this happening a lot?

Reply:
I saw it quite a few times and I fear it might have happened a lot more.
Also those who ran into this had no clue why this could be the source of their problem.

After all it is not general knowledge that e.g. `@testing-library/react` when imported has the side-effect to change behavior in `@testing-library/dom`. (I think others frameworks might do similar things.)

But previously the async wrapper was used sparely because most people didn't use the `delay` option.
Now all APIs are asynchronous.